examples/simple_example/main.tf

```diff
@@ -68,6 +74,14 @@ module "healthcare" {
       notification_config = {
         pubsub_topic = local.pubsub_topic
       }
+      stream_configs = [{
```

Comment:
If there is gcloud commands available for stream configs, could you also follow the pattern in https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/test/integration/simple_example/controls/gcloud.rb to add some gcloud commands to check that the stream configs is successfully configured?

Reply:
Definitely possible. That said, I think there are many gaps in the current test coverage. If this is different than checking for notification/pubsub configuration (which doesn't appear to be done currently) then I think it makes sense to add these checks in this PR. If not, I think we can leave it out.

Do you think we should file an issue to make more granular tests? The current tests don't check for permutations of optional resource configurations which I think is a bigger risk.

Reply:
There were some tests for notification configs before, but got removed in https://github.com/terraform-google-modules/terraform-google-healthcare/pull/16/files. Checking notification configs and stream configs should be pretty similar. You can leave them out for this PR but open an Issue to add those.

Reply:
Opened: https://github.com/terraform-google-modules/terraform-google-healthcare/issues/23